src/avro/tests/io.rs

```diff
@@ -286,7 +288,7 @@ fn test_binary_long_encoding() {
 fn test_schema_promotion() {
     // Each schema is present in order of promotion (int -> long, long -> float, float -> double)
     // Each value represents the expected decoded value when promoting a value previously encoded with a promotable schema
-    let promotable_schemas = vec![r#""int""#, r#""long""#, r#""float""#, r#""double""#];
+    let promotable_schemas = [r#""int""#, r#""long""#, r#""float""#, r#""double""#];
```

Comment:
Just curious what's going on here -- why'd it tell you not to use a vec for `promotable_schemas`, but it was fine with `promotable_values` ?